### PR TITLE
add function to include external stylesheets

### PR DESCRIPTION
**Pull request by Roywcm**
I needed to include external stylesheets, so I made a function for that. What do you think?
I took some code from https://github.com/tijsverkoyen/CssToInlineStyles/issues/95

### CODE REVIEW

File: src/CssInlinerPlugin.php [hunk 1]
```diff
@@ -33,6 +35,7 @@ public function beforeSendPerformed(\Swift_Events_SendEvent $evt)
         }
     }
 
+
```
Comment (GrahamCampbell) [inline src/CssInlinerPlugin.php]: ?

File: src/CssInlinerPlugin.php [hunk 2]
```diff
@@ -12,8 +12,9 @@ class CssInlinerPlugin implements \Swift_Events_SendListener
     public function beforeSendPerformed(\Swift_Events_SendEvent $evt)
     {
         $message = $evt->getMessage();
-
+
```
Comment (GrahamCampbell) [inline src/CssInlinerPlugin.php]: ?

File: src/ExternalCssPlugin.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        
+        $cssContent = @file_get_contents($absUrl);
+
+        if (!$cssContent) {
+            trigger_error('Error loading stylesheet from ' . $absUrl.'--'.getcwd(), E_USER_NOTICE);
+        }
+
+        return $cssContent;
+    }
+}
```
Comment (GrahamCampbell) [inline src/ExternalCssPlugin.php]: cs

File: tests/ExternalCssPluginTest.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    /** @test **/
+	public function getHtmlBody(){
+		
+		$external = new ExternalCssPlugin();
+		$orginalHtml = str_replace('%styleUrl%', __DIR__.'/stubs/externalStyle.css', $this->stubs['original-html']);
+		$newHtml = $external->includeExternalStylesheets($orginalHtml);
+
+		$this->assertEquals($this->stubs['converted-html'],$newHtml);
+	}
+}
```
Comment (GrahamCampbell) [inline tests/ExternalCssPluginTest.php]: cs

**Comment by Roywcm** [comment]
Made an external Laravel plugin for this https://github.com/Roywcm/laravel-css-includer

**Pull request closed without merging**